CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ target_link_libraries(acm pthread rdkafka++ asncodec pugixml)
 target_link_libraries(acm_tests pthread rdkafka++ asncodec pugixml Catch)
 target_compile_definitions(acm_tests PRIVATE _ASN1_CODEC_TESTS) 
 
+add_subdirectory(kafka-test)
```

Comment:
I've been wondering why is this called `docker-test`? Shouldn't this be `docker-run`? test implies that the directory contains test files. But clearly the main script that launches the application is not just for testing. I know it's just a name but it is a bit misleading, don't you think?